explorerlib/indexCrawler.go

```diff
@@ -39,14 +54,21 @@ func FindWordsByLanguageInIndex(index Index, fields []string, documentsExplorati
 				randomWord := ""
 
 				for i := 0; i < queryNumber; i++ {
+
 					// build A query from the word counts in the appropriate language with a filter on the field value
 					queryExpression := randomWord +
 						" @syslanguage=\"" + language + "\" " +
 						field + "=\"" + value.Value + "\" "
+
+					t1 = time.Now()
```

Comment:
It does not work, dt1 is updated many times in the inner loop. It should not be the same variable as the first loop.